config/routes.rb

```diff
@@ -74,6 +74,7 @@
 
     get "/backdate" => "backdate#edit", as: :backdate
     post "/backdate" => "backdate#update"
+    delete "/backdate" => "backdate#destroy", as: :destroy_backdate
```

Comment:
I don't think we need to set a path name for this.

Reply:
Removed